config/initializers/ruby_stackoverflow.rb

```diff
@@ -1,5 +1,4 @@
 RubyStackoverflow.configure do|config|
-  config.client_key = ''
-  config.access_token = ''
+  config.client_key = 'ZEGYIdaRKJ4eFpqjOrX6dw(('
+  config.access_token = 'h5Q5iW2AAJSr0n4bB7(shA))'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.